skyportal/handlers/api/photometry.py

```diff
@@ -949,14 +943,11 @@ def delete(self, upload_id):
               application/json:
                 schema: Error
         """
-
-        # dont check permissions here -- pull all the photometry associated with
-        # the upload, not necessarily just the photometry that is accessible
-        # to the user. if any of the photometry fails to be deleted, send back
-        # 400
-        photometry_to_delete = Photometry.query.filter(
-            Photometry.upload_id == upload_id
-        ).all()
```

Comment:
i think we want to keep this as is - so that if a user requests that we delete photometry ids [1, 4, 7], but they don't have access to 4, then the whole transaction rolls back, instead of deleting just 1 and 7 and silently keeping 4

Reply:
Is that even possible for a given `upload_id`? Given that `delete = accessible_by_owner`?